store/unit-slicer.js

```diff
@@ -77,7 +77,11 @@ export const unitSlice = createSlice({
     },
 
     setStandards: (state, action) => {
-      state.details.standards.push(...action.payload);
+      state.details.standards.shift(...action.payload);
```

Comment:
I think you meant unshift here. When I tried to add standards it ended up deleting them instead.

Reply:
@blakes24, oh yea, major typo!  I updated this and also added a check to avoid adding standards if there were no selected standards.

Reply:
👍